manifests/t/TrackerSoftware/PDF-XChangeViewer/2.5.322.10/TrackerSoftware.PDF-XChangeViewer.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TrackerSoftware.PDF-XChangeViewer
+PackageVersion: 2.5.322.10
+PackageLocale: en-US
+Publisher: Tracker Software Products Ltd

```

```suggestion
Publisher: Tracker Software Products (Canada) Ltd.
```
This must match Control Panel.